website/screens/principles/color/code/examples/internalUse.js

```diff
@@ -1,23 +1,37 @@
 import {
-  DxcInset,
+  HalstackProvider,
   DxcTextInput,
-  DxcFooter,
+  DxcInset,
+  DxcAccordion,
 } from "@dxc-technology/halstack-react";
 
 const code = `() => {
+  const customTheme = {
+    accordion: {
+      backgroundColor: "#000000",
+      hoverBackgroundColor: "#111111",
+      arrowColor: "#ffffff",
+      titleLabelFontColor: "#ffffff",
+    },
+  };
   return (
-    <DxcInset space="2rem">
-      <DxcFooter>
-        <DxcTextInput label="Enter your surname" defaultValue="Harris" />
-      </DxcFooter>
-    </DxcInset>
+    <HalstackProvider advancedTheme={customTheme}>
+      <DxcInset space="1rem">
+        <DxcAccordion isExpanded label="Accordion">
```

Comment:
Please keep inset space as `2rem`, to keep it as before and equals to the custom use example.